src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10690,6 +10780,22 @@ definitions:
         $ref: '#/definitions/PermissionsInfo'
       definition:
         $ref: '#/definitions/Definition'
+  calculateSizeResponse:
+    type: object
+    properties:
+      id:
+        type: string
+      size:
+        type: string
+      calculatedAtTime:
+        type: string
+      status:
+        type: string
```

Comment:
It's a good practice to add description for all the fields.

Reply:
Done